gridcapa-core-cc-post-processing-app/src/main/java/com/farao_community/farao/core_cc_post_processing/app/services/DailyF303Generator.java

```diff
@@ -43,32 +46,38 @@ public DailyF303Generator(MinioAdapter minioAdapter) {
     }
 
     public FlowBasedConstraintDocument generate(Map<TaskDto, ProcessFileDto> raoResults, Map<TaskDto, ProcessFileDto> cgms) {
-        String cracFilePath = raoResults.keySet().stream()
+        ProcessFileDto cracFile = raoResults.keySet().stream()
```

Comment:
Why did you change here to get a DTO instead of the path (which is the only element of the DTO you will use in the next lines of this method)?